### ISSUE TITLE
Can ipex support general llm like defog/sqlcoder

### ISSUE THREAD

**Issue by RenyanDiao** [intel/intel-extension-for-pytorch#413]
### Describe the issue

 model = ipex._optimize_transformers(model.eval(), dtype=torch.bfloat16, inplace=True) 

![image](https://github.com/intel/intel-extension-for-pytorch/assets/52916011/8d2a7d0f-3fac-4eb0-b910-b99757fec15a)

**Comment by jingxu10** [comment]
No, currently only models mentioned in the screenshot are optimized.

**Comment by RenyanDiao** [comment]
> No, currently only models mentioned in the screenshot are optimized.

Do you have plans to support more models recently?

**Comment by jingxu10** [comment]
Is there any reproducer that we can use to investigate this crash issue?
Models outside the list should be able to run, but will not be optimized.

**Comment by RenyanDiao** [comment]
> Is there any reproducer that we can use to investigate this crash issue? Models outside the list should be able to run, but will not be optimized.

https://github.com/defog-ai/sqlcoder/tree/main
yes, it can run while just performs as the same as using official torch. The latency is too long for us.
